scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,96 @@ spec: cssom-view-1; type: dfn;
 
 	A set of animation keyframes can be attached
 	in reference to [=named timeline ranges=]
-	by adjusting the duration of the animation.
+	by restricting the animation’s [=active phase=]
+	to the specified timeline range.
+	Delays (see 'animation-delay')
+	are set within this restricted range,
+	constricting it further.
+	If a duration ('animation-duration') is given,
+	it extends from the end of the start delay
+	until the start of the end delay,
+	and further restricts the range-restricted end.
+
+	<pre class="ascii-art">
+	range start┐             ╺┉┉┉┉active phase┉┉┉┉╸           ┌range end
+	┄┄┄┄┄┄┄┄┄┄┄├─────────────╊━━━━━━━━━━━━━━━━━━━━╉───────────┤┄┄┄┄┄┄┄┄
+               ╶┄start delay┄╴                    ╶┄end delay┄╴
+                             ╶┄┄┄┄┄ duration┄┄┄┄┄┄╴
```

Comment:
I think this diagram might have some alignment issues, this is what I'm seeing locally when viewing the spec:
![Screenshot 2023-02-02 at 3 19 44 PM](https://user-images.githubusercontent.com/366761/216440932-e4aee15b-c060-41b0-a11c-75d00f4ed50f.png)
The start delay, duration and end delay are meant to line up exactly with the ranges and active phases I assume.

Reply:
Tabs vs spaces issue. :/ Fixed now.